scripts/components/content.js

```diff
@@ -44,7 +44,7 @@ const content_frame = new Vue({
     },
     all_songs(){
       console.log("Updated")
-      return Object.values(state.songsMap)
+      return Object.values(state.songsMap).sort() //Now we have sorted list of all_songs
```

Comment:
A better solution will be to store the list already sorted in cache rather than sorting everytime when we launch the application.